mp/src/game/client/momentum/ui/HUD/hud_stickybombs.cpp

```diff
@@ -1,20 +1,64 @@
 #include "cbase.h"
 
+#include <vgui_controls/AnimationController.h>
 #include <vgui_controls/EditablePanel.h>
-#include <vgui_controls/Label.h>
 
 #include "c_mom_player.h"
 #include "hudelement.h"
 #include "iclientmode.h"
 #include "mom_system_gamemode.h"
 #include "weapon/weapon_mom_stickybomblauncher.h"
+#include "weapon/weapon_shareddefs.h"
 
 #include "tier0/memdbgon.h"
 
+#define STICKY_COUNT_ARM_ANIM_NAME "FadeInStickyArm"
+
 using namespace vgui;
 
 static MAKE_TOGGLE_CONVAR(mom_hud_sj_stickycount_enable, "1", FCVAR_ARCHIVE, "Toggles the stickybomb counter.\n");
-static MAKE_TOGGLE_CONVAR(mom_hud_sj_stickycount_autohide, "0", FCVAR_ARCHIVE, "Toggles automatically hiding the stickybomb counter at 0 stickies. 0 = OFF, 1 = ON\n");
+
+class Stickybox : public Panel
+{
+    DECLARE_CLASS_SIMPLE(Stickybox, Panel);
+
+  public:
+    Stickybox(Panel *pParent);
+
+    bool HasAnimationStarted() const { return m_bAnimationStarted; }
+
+    void StartAnimation(Color startColor);
+    void StopAnimation();
+
+    void OnThink() override;
+
+  private:
+    CPanelAnimationVar(Color, m_StickyColor, "StickyColor", "BlackHO");
+
+    bool m_bAnimationStarted;
+};
+
+Stickybox::Stickybox(Panel *pParent) : BaseClass(pParent, "Stickybox"), m_bAnimationStarted(false)
+{}
+
+void Stickybox::StartAnimation(Color startColor)
+{
+    m_StickyColor = startColor;
+    vgui::GetAnimationController()->StartAnimationSequenceForPanel(this, STICKY_COUNT_ARM_ANIM_NAME);
```

Comment:
You are using namespace vgui so you can remove `vgui::` from this.

Reply:
i took that change request a little too literally 😑